specification/dartLangSpec.tex

```diff
@@ -8848,53 +8851,84 @@ \subsubsection{Unqualified Invocation}
 }
 
 \LMHash{}%
-It is a compile-time error if $i$ occurs inside a top-level or static function
-(be it function, method, getter, or setter)
-or a top-level or static variable initializer,
-and there is no lexically visible declaration named \id{} in scope.
+For the static analysis of $i$,
```

Comment:
I'm not particularly fond of starting a sentence with a ... preposition phrase? (Is that a thing).

Maybe just:

> First perform a lexical lookup of \id{} (\ref{lexicalLookup}) from the location of $i$.

(A lexical lookup needs a context, you can't do "lexical lookup" in the abstract).